### PR TITLE
add details-Page

### PR DESCRIPTION
**Pull request by magjo21**

### CODE REVIEW

**Review by doemser — CHANGES_REQUESTED** [review]
Looks good! :)

Only hat a few comments. The SVGIcon component should be in its own file and imported from there wherever it is needed in your project.

File: pages/details/[type].js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                {aspect.name}: {aspect.value}
+              </li>
+            );
+          })}
+        </ul>
+        <Link href={"/overview"}>Go back</Link>
+      </>
+    );
+  }
+}
```
[the remark this answers is not in the document]
Comment (magjo21) [reply pages/details/[type].js:27]: You are right I will take it out

File: pages/overview.js [hunk 2]
```diff
@@ -1,34 +1,33 @@
...
     <>
       <h1>Overview</h1>
-      <p>Here comes the details page.</p>
-      <SVGIcon
-        name="happy emoticon"
-        variant="happy"
-        witdh="50px"
-        color="green"
-      />
+      <p>Here comes the details page and it should look litke this.</p>
```
[the remark this answers is not in the document]
Comment (magjo21) [reply pages/overview.js:12]: You are right I will take it out

**Review by niklasrammelkamp — APPROVED** [review]
great job !

**Pull request merged**